util/logutil/slow_query_logger.go

```diff
@@ -13,11 +13,11 @@ import (
 
 var _pool = buffer.NewPool()
 
-func newSlowQueryLogger(cfg *LogConfig) (*zap.Logger, error) {
+func newSlowQueryLogger(cfg *LogConfig) (*zap.Logger, *log.ZapProperties, error) {
```

Comment:
Why add the `*log.ZapProperties`? It seems that we don't use it.

Reply:
Those props are used in `TestSlowQueryLoggerCreation` function.